src/pages/index.astro

```diff
@@ -123,6 +124,25 @@ posts
       </Container>
     </div>
   </div>
+  <div>
```

Comment:
The [donate page](https://nixos.org/donate/) clarified that “following companies and institutes have supported the development of NixOS and hosting for its infrastructure”. Here it is not clear what the logos mean. Landing pages often show users or endorsements so it might be important to the sponsors themselves for the relationship to be made clear.